### PR TITLE
Show collection name in question tag autocomplete results

### PR DESCRIPTION
**Pull request by calherries**
Shows the collection name in question tag autocomplete results in the native query editor.

Part of [this epic](https://github.com/metabase/metabase/issues/25050).

Before:
image

After:
image

### CODE REVIEW

**Review by gusaiani — APPROVED** [review]
👍

File: src/metabase/api/database.clj [hunk 1]
```diff
@@ -544,7 +545,8 @@
   (try
     (->> (autocomplete-cards id query)
          (filter mi/can-read?)
-         (map #(select-keys % [:id :name :dataset])))
+        ;;  ((fn [xs] (prn (map mi/can-read? xs)) xs))
```
Comment (noahmoss) [inline src/metabase/api/database.clj:548]: did you mean to leave this in?

**Review by noahmoss — APPROVED** [review]

File: src/metabase/api/database.clj (hunk 1 above)
Comment (calherries) [reply src/metabase/api/database.clj:548]: No, thanks for catching

**Comment by mazameli** [comment]
Right now it looks like the max-width is ~400px for this popover, and I think we could increase it to a max of 520px to allow more of the question and collection names to fit.

image

**Comment by mazameli** [comment]
Is it just my imagination or did we used to show the collection that the model/saved question is saved in here in the sidebar? 

image

image

In any case, we should to that, like the mock in the product doc:

image

**Comment by calherries** [comment]
@mazameli 
> Is it just my imagination or did we used to show the collection that the model/saved question is saved in here in the sidebar?

That's in a PR ([link](https://github.com/metabase/metabase/pull/26063)) I haven't actually merged to master yet. I'm still to resolve a type issue.

**Review by mazameli — APPROVED** [review]

**Pull request merged**